### PR TITLE
Playlist fix

### PR DESCRIPTION
**Pull request by Developers-codz**
fixed duplicacy of playlist name, added disabled button functionality when API call is going, fixed single video page bug now app don't crash on refresh in single video page

### CODE REVIEW

**Review by simran61 — APPROVED** [review]
Looks good 
Clean code

File: src/components/cards/SingleCard.jsx [hunk 1]
```diff
@@ -3,13 +3,19 @@ import { useState } from "react";
...
 import { useDeleteFunction } from "functions";
+import { useLike } from "context/like-context";
+import { useWatchLater } from "context/watchlater-context";
+import { usePlaylist } from "context/playlist-context";
 
 export const SingleCard = ({ item, flag }) => {
   const [showMenu, setShowMenu] = useState(false);
   const getFunctionName = useDeleteFunction();
   const clickHandler = () => {
     setShowMenu(!showMenu);
```
Comment (Dev-AkshatGupta) [inline src/components/cards/SingleCard.jsx]: Suggestion you should have wrote a call back only where there is the click handler and also you should have used callback inside the setShowMenu

File: src/components/navbar/navbar.module.css [hunk 2]
```diff
@@ -43,6 +43,10 @@
     font-size: xx-large;
   }
   .searchBoxWrapper{
-    display: none;
+    order:10;
```
Comment (Dev-AkshatGupta) [inline src/components/navbar/navbar.module.css:46]: Nive that you are using the order

File: src/page/playlist/Playlist.jsx [hunk 3]
```diff
@@ -44,13 +44,15 @@ export const Playlist = () => {
...
                       navigate("/playlist");
-                    }}
+                    }}>
+
+                  <i
+                    className="fa fa-trash"
                   ></i>
+                  </button>
                   <div className={styles.showPlaylistnum}>
                     <h1>+{videos.length}</h1>
```
Comment (Dev-AkshatGupta) [inline src/page/playlist/Playlist.jsx:57]: Suggestion : Take care of indentation.

File: src/page/playlist/playlist.module.css [hunk 4]
```diff
@@ -60,9 +60,16 @@
...
+
+.delPlaylistBtn{
+  background-color: transparent;
+  color:var(--COLOR-DEFAULT-TEXT);
   z-index: 3;
   cursor: pointer;
+  border:none;
+}
+.delPlaylistBtn :disabled{
+cursor: not-allowed;
```
Comment (Dev-AkshatGupta) [inline src/page/playlist/playlist.module.css:72]: Indentation

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything seems great I have added the needed suggestions.

File: src/components/cards/SingleCard.jsx (hunk 1 above)
Comment (Developers-codz) [reply src/components/cards/SingleCard.jsx]: That's a click handler and I wanted to write the CB function separate, And for callback in setShowMenu I will do that

File: src/components/navbar/navbar.module.css (hunk 2 above)
Comment (Developers-codz) [reply src/components/navbar/navbar.module.css:46]: Thank you

File: src/page/playlist/Playlist.jsx (hunk 3 above)
Comment (Developers-codz) [reply src/page/playlist/Playlist.jsx:57]: yep

**Pull request merged**